docs/src/Checkbox.doc.js

```diff
@@ -85,64 +93,35 @@ card(
     name="Example: Accessibility"
     defaultCode={`
 function CheckboxExample() {
-  const [checked, setChecked] = React.useState(true);
+  const [checkedSm, setCheckedSm] = React.useState(true);
+  const [checkedMd, setCheckedMd] = React.useState(true);
+
   return (
-    <Box alignItems="center" direction="row" display="flex">
+    <Box margin={-2}>
+      <Box padding={2}>
       <Checkbox
-        checked={checked}
-        id="usa"
-        name="usa"
+        checked={checkedSm}
+        id="usaSm"
+        label="United States of America"
+        name="usaSm"
         onChange={({ checked }) => {
-          setChecked(checked);
+        setCheckedSm(checked);
```

Comment:
When looking at the example, the formatting looks off: 
![image](https://user-images.githubusercontent.com/127199/76996605-0a82ed80-690f-11ea-95c9-a384206927fc.png)

I usually run the code though prettier and then copy/paste it in to the example

Reply:
Gotcha!

Reply:
DONE